lms/templates/email/instructor_email_digest/subject.jinja2

```diff
@@ -1,11 +1,11 @@
 {% if total_annotations > 1 and annotators|length > 1 and courses|length > 1 -%}
-Hypothesis: {{ total_annotations }} annotations from {{ annotators|length}} students in {{ courses|length }} courses
+Hypothesis: {{ total_annotations }} new annotations from {{ annotators|length}} students in {{ courses|length }} courses
 {%- elif annotators|length > 1 -%}
-Hypothesis: {{ total_annotations }} annotations from {{ annotators|length}} students in 1 course
+Hypothesis: {{ total_annotations }} new annotations from {{ annotators|length}} students in 1 course
 {%- elif courses|length > 1 -%}
-Hypothesis: {{ total_annotations }} annotations from 1 student in {{ courses|length }} courses
+Hypothesis: {{ total_annotations }} new annotations from 1 student in {{ courses|length }} courses
 {%- elif total_annotations > 1 -%}
-Hypothesis: {{ total_annotations }} annotations from 1 student in 1 course
+Hypothesis: {{ total_annotations }} new annotations from 1 student in 1 course
 {%- else -%}
-Hypothesis: one of your students made an annotation
+Hypothesis: one of your students made a new annotation
```

Comment:
One of the tweaks to the email wording also applies to subjects: saying "new annotation(s)" instead of just "annotation(s)"